apps/protocol-frontend/src/libs/ipfs.ts

```diff
@@ -9,15 +14,14 @@ export const getIPFSClient = () => {
         ':' +
         import.meta.env.VITE_INFURA_PROJECT_SECRET,
     ).toString('base64');
-  const ipfs = create({
+  return create({
```

Comment:
Nice. I like this -- I'll start following this pattern too instead of using the unnecessary variable name